DEVELOPMENT.md

```diff
@@ -33,13 +33,25 @@ $ git remote add ${USER} https://github.com/${USER}/build
 
 Configure `ko` to point to your registry:
 
+For a gcr.io project (like "myproject"), it will need to have Container
+Registry API enabled first.
 ```shell
 # You can put these definitions in .bashrc, so this is one-time setup.
 #
 # KO_DOCKER_REPO is the image repository where the images for build
 # will be pushed, including hostname.
-export KO_DOCKER_REPO=us.gcr.io/project
+export KO_DOCKER_REPO=us.gcr.io/myproject
```

Comment:
Can you also `s/myproject/<myproject>/` so that this fails early during parsing instead of at authentication time?

Reply:
good thought. done.